config/system.php

```diff
@@ -407,6 +407,14 @@
 'default_value'=>'y'
 );
 
+$system__options_general[]=array(
+'option_name'=>'include_weekday_in_session_name',
```

Comment:
Ben, please add indent for each new line within array(), otherwise difficult to follow. We can make this as rule for coding style if you do not mind, e.g.
```
$array = array(
    'option1' => 'value1',
    'option2' => 'value2',
);
```